src/client/mixins/basic-interaction.ts

```diff
@@ -205,7 +219,7 @@ export class BasicInteractionAware {
             .catch(e => rej(Error('Submit button still there')));
         }),
         new Promise((res, rej) => {
-          this.client.click(selector)
+          this.client['___currentFrame'].click(selector)
             .then(res)
             .catch((e) => {
               this.client.waitForFunction(
```

Comment:
- [ ] One missing `['___currentFrame']` reference.

Reply:
Oh I already fixed that beforehand since my unit tests failed.

Reply:
Fixing the `dom.ts` mixin for this comment